plugins/tutor-contrib-paragon/README.rst

```diff
@@ -0,0 +1,63 @@
+Paragon plugin for `Tutor <https://docs.tutor.edly.io>`__
+###########################################################
+
+Facilitates the generation and static hosting of Paragon-based CSS themes for Open edX Micro-Frontend (MFE) applications using `Paragon <https://openedx.github.io/paragon/>`__.
+
+This plugin provides a local folder structure to manage theme source files and compile them into CSS, enabling flexible customization of Open edX MFEs via Tutor.
+
+Installation
+************
+
+.. code-block:: bash
+
+    pip install git+https://github.com/openedx/openedx-tutor-plugins.git#subdirectory=plugins/tutor-contrib-paragon
+
+For development:
+
+.. code-block:: bash
+
+    cd openedx-tutor-plugins/plugins/tutor-contrib-paragon
+    pip install -e .
+
+Enable the plugin:
+
+.. code-block:: bash
+
+    tutor plugins enable paragon
+
+Directory Structure
+*******************
+
+The plugin will create the following structure inside your Tutor environment:
+
+.. code-block::
+
+    tutor/env/plugins/paragon/
+    ├── theme-sources/           # Place your style dictionary folders here (e.g. theme-xyz/)
+    └── compiled-themes/         # Compiled CSS files go here, ready for static hosting
```

Comment:
When site operators start converting their existing themes for Paragon 23 I expect a _lot_ of the styles to work as design tokens but not all. I'd like to think through that use case:
* Site operator converts all Paragon scss variable based branding to use design tokens
  * This works perfectly with the `theme-sources` and `compiled-themes` directory structure
* Site operator then wants to add a small amount of custom CSS
  * It's not clear what the recommended way to do this is. Compile the tokens and then add things by manually editing the compiled css?